projects/helper/tokenMapping.js

```diff
@@ -65,7 +65,7 @@ const fixBalancesTokens = {
     USDC: { coingeckoId: 'usd-coin', decimals: 0 },
     USD: { coingeckoId: 'usd-coin', decimals: 0 },
     YLDS: { coingeckoId: 'usd-coin', decimals: 0 },
-    'pm.sale.pool.3dxq3fk9llvhrqqwhodiap': { coingeckoId: 'usd-coin', decimals: 0 },
```

Comment:
why is this removed?

Reply:
We effectively replaced this by adding FIGR_HELOC.